Mantle/MTLJSONAdapter.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  MTLJSONAdapter.swift
+//  Mantle
+//
+//  Created by Zachary Waldowski on 1/21/16.
+//  Copyright © 2016. Some rights reserved. Licensed under MIT.
+//
+
+public enum MTLJSONAdapterError: ErrorType, CustomStringConvertible {
```

Comment:
Should this use [`_BridgedNSError`] instead? https://github.com/apple/swift/blob/master/stdlib/public/SDK/Foundation/NSError.swift#L126